runfolder/handlers.py

```diff
@@ -1,7 +1,13 @@
+
+import tornado.web
+
+
 import arteria
+from arteria.web.state import State
```

Comment:
Well - on the other hand one could argue that renaming with e.g. a prefix to `ArteriaState` would complicate things unnecessarily, and if we have a name collision we can always use `... import State as ArteriaState`. What do you think?